react_app/src/App.js

```diff
@@ -1,35 +1,156 @@
+import Clock from "./Clock";
+import Card from './Cards.js'
+import React, { useState} from "react";
+
+const api = {
+  key: "af103c190cd36ff3f3fb1e0c135a2ee1",
+  base: "https://api.openweathermap.org/data/2.5/"
+}
 
-import { Component } from 'react';
-import Navbar from "./Components/Navbar/Navbar";
-import { Jumbotron } from 'reactstrap';
-import './App.css';
 
-let apikey = "af103c190cd36ff3f3fb1e0c135a2ee1";
 
+function App() {
 
-class App extends Component{
+  const [query, setQuery] = useState('');
+  const [weather, setWeather] = useState({});
+  const [weather2, setWeather2] = useState({});
   
-  // constructor(props){
-  //   super(props);
-  //   this.state={
-  //     fade1 : false,
-  //     fade2 : false
-  //   }
-  // }
-  render(){
-    let cname = "lead";
-    return(
-      <div>
-        <Navbar />
-        <Jumbotron>
-          <h1 className = "display-3">Weather app stuff</h1>
-          <p className = {cname}> This is my first Weather App</p>
-          <hr className = {"my-2"}/>
-          <p className = {cname}></p>
-        </Jumbotron>
-      </div>
-    )
+  const search = evt => {
+    //if (evt.key === "Enter") {
+    fetch(`${api.base}weather?q=${query}&units=imperial&APPID=${api.key}`)
+      .then(res => res.json())
+      .then(result => {
+        setWeather(result);
+        // setQuery('');
+        console.log(result);
+        if (result.cod === 200) {
+          fetch(`${api.base}onecall?lat=${result.coord.lat}&lon=${result.coord.lon}&units=imperial&appid=${api.key}`)
+            .then(res2 => res2.json())
+            .then(result2 => {
+              console.log(result2)
+              setWeather2(result2);
+              setQuery('');
+            })
+        }
+      });
+    //}
+  }
+
+  const dateBuilder = (d) => {
+    // let days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+    let months = ["January", "February", "March", "April", "May", "June", "July",
+      "August", "September", "October", "November", "December"];
+
+    // let day = days[d.getDay()];
+    let date = d.getDate();
+    let month = months[d.getMonth()];
+    let year = d.getFullYear();
+
+
+    return `${date} ${month} ${year}`
+
+
   }
+
+  function toTitleCase(str) {
+    return str.replace(/\w\S*/g, function (txt) {
+      return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+    });
+  }
+
+  return (
+    <div className={(typeof weather.main != "undefined") ?
+      (weather.weather[0].main === "Clouds" ? 'App clouds' :
```

Comment:
instead of all these ternarys since your class name matches the condition use
(typeof weather.main != "undefined") ? `App ${weather.weather[0].main && weather.weather[0].main.toLowerCase()}` : 'App';